Wrapping/Python/vtk/qt4/QVTKRenderWindowInteractor.py

```diff
@@ -218,11 +217,27 @@ def Finalize(self):
         self._RenderWindow.Finalize()
 
     def CreateTimer(self, obj, evt):
-        self._Timer.start(10)
+        timer_id = self.GetTimerEventId()
+        timer_duration = self.GetTimerDuration(timer_id)
+        timer_type = self.GetTimerEventType()
+
+        t = QtCore.QTimer()
+        t.connect(t, QtCore.SIGNAL("timeout()"), self.TimerEvent)
+        if timer_type == self.OneShotTimer:
+            t.setSingleShot(True)
+        else:
+            t.setSingleShot(False)
+        t.start(timer_duration)
+        self._timers[timer_id] = t
```

Comment:
Is that not how timers work in VTK? Looking in [vtkRenderWindowInteractor](https://github.com/Kitware/VTK/blob/640d322bd5ad2fd2748ea0ffecfa5176d48403a1/Rendering/Core/vtkRenderWindowInteractor.cxx#L481-L501), it looks like I'm taking roughly the same approach, creating a map of timer ids to timers. I use timers extensively throughout the UI code as well as the animation code, and will frequently call `DestroyTimer` to cancel events. The way that this was behaving was canceling all currently running timers as well, which would tend to break a lot of things.